api/src/main/java/org/openmrs/Concept.java

```diff
@@ -449,7 +449,7 @@ public ConceptName getName(Locale locale) {
 	public ConceptName getName() {
 		if (getNames().isEmpty()) {
 			if (log.isDebugEnabled()) {
```

Comment:
Now that you are using the parametrized message style you should also remove the `if (log.isDebugEnabled())`. Please do this in all places that you changed.